assets/stylesheets/custom.scss

```diff
@@ -3,235 +3,253 @@ $red: #ae442e;
 $orange: orange;
 
 .good {
-	color: rgba(0, 255, 0, 0.4);
+  color: rgba(0, 255, 0, 0.4);
 }
 
 .bad {
-	color: rgba(255, 0, 0, 0.4);
+  color: rgba(255, 0, 0, 0.4);
 }
 
 h1 {
-	font-size: 22px;
+  font-size: 22px;
 }
 
 h1.header {
-	color: rgba(255,255,255,0.5);
-	font-weight: bold;
-	font-size: 30px;
+  color: rgba(255, 255, 255, 0.5);
+  font-weight: bold;
+  font-size: 30px;
 }
 
 h2 .value-add {
-	font-size: 0.4em;
-	vertical-align: middle;
+  font-size: 0.4em;
+  vertical-align: middle;
 }
 
 .trend-up {
-	color: $green;
+  color: $green;
 }
 
 .trend-down {
-	color: $red;
+  color: $red;
 }
 
 .more-info {
-	font-size: 12px;
-	opacity: 0.5;
+  font-size: 12px;
+  opacity: 0.5;
 }
 
 .highcontrast {
 
-	background-color: #222;
+  background-color: #222;
 
-	.widget {
-		color: white;
-		background-color: #333 !important;
-		padding: 0;
+  .widget {
+    color: white;
+    background-color: #333 !important;
+    padding: 0;
 
-		.updated-at {
-			color: rgba(255,255,255,0.5);
-		}
+    .updated-at {
+      color: rgba(255, 255, 255, 0.5);
+    }
 
-		.x_tick .title {
-			color: rgba(255,255,255,0.5);
-			// Avoid overlap with cols, a bit hacky
-			padding-bottom: 15px;
-			padding-left: 10px;
-		}
+    .x_tick .title {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 4<br>Selector `x_tick` should be written in lowercase with hyphens